rush/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -15,6 +15,16 @@ import Utilities from '../utilities/Utilities';
 import { RushConstants } from '../RushConstants';
 
 /**
+ * A list of known config filenames that are expected to appear in the "./common/rush/config" folder.
```

Comment:
Fixed -- this is just a typo in the comment

---
In reply to: [115864530](https://github.com/Microsoft/web-build-tools/pull/176#discussion_r115864530) [](ancestors = 115864530)